src/deepsparse/engine.py

```diff
@@ -131,6 +132,21 @@ def _validate_scheduler(scheduler: Union[None, str, Scheduler]) -> Scheduler:
     return scheduler
 
 
+class Context(object):
```

Comment:
can you put a block comment describing the function of Context? i'm thinking similar to the Scheduler